library/arangodb

```diff
@@ -5,8 +5,8 @@
 2.8.11: https://github.com/arangodb/arangodb-docker@d6fca9a137cd21345b1d380fc0e72daacb6130ce jessie/2.8.11
 3.2: https://github.com/arangodb/arangodb-docker@b4c22ad8bf4facbbc7c3b24e985251a09fcdbcec stretch/3.2.17
 3.2.17: https://github.com/arangodb/arangodb-docker@b4c22ad8bf4facbbc7c3b24e985251a09fcdbcec stretch/3.2.17
-3.3: https://github.com/arangodb/arangodb-docker@c77fa2b9bca45deb37e2a709dead1147541857ba stretch/3.3.21
-3.3.21: https://github.com/arangodb/arangodb-docker@c77fa2b9bca45deb37e2a709dead1147541857ba stretch/3.3.21
 3.4: https://github.com/arangodb/arangodb-docker@1440bf4bf9258df24a794c92e43ca586e62e4bb2 alpine/3.4.4
 3.4.4: https://github.com/arangodb/arangodb-docker@1440bf4bf9258df24a794c92e43ca586e62e4bb2 alpine/3.4.4
 latest: https://github.com/arangodb/arangodb-docker@1440bf4bf9258df24a794c92e43ca586e62e4bb2 alpine/3.4.4
+3.3: https://github.com/arangodb/arangodb-docker@dcc90263a2247e5dba847c7dd4fc30197a191539 stretch/3.3.23
+3.3.23: https://github.com/arangodb/arangodb-docker@dcc90263a2247e5dba847c7dd4fc30197a191539 stretch/3.3.23
```

Comment:
I don't see a `3.3.23` in your repo at this commit. This should probably be `3.3.22` and both should be the folder `3.3.22`

Reply:
cc @dothebart @fceller